playbook/guide-staging.yml

```diff
@@ -0,0 +1,33 @@
+#
+# This playbook installs a docker, composes and keeps it running.
+# Then it fetches pointed in the host an image of Guide from quay.io,
+# runs a container, and starts the Guide.
+# And all of them will restarted if a server rebooted.
+#
+# Launch sites on http not https.
```

Comment:
What does this mean?

Reply:
http://staging.guide.aelve.com:8080/ launched
https://staging.guide.aelve.com:8080/ not launched